### PR TITLE
a11y: change page title

### PR DESCRIPTION
**Pull request by mahdikhashan**
addresses part of # 414 

https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html

### CODE REVIEW

File: boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/controller/StartPageController.kt [hunk 1]
```diff
@@ -14,7 +14,7 @@ import org.springframework.web.servlet.ModelAndView
 @RequestMapping("/")
 class StartPageController @Autowired constructor(private val eventService: EventService) {
 
-    private val PAGE_TITLE = "Boudicca"
+    private val PAGE_TITLE = "Boudicca.Events - Open source event collector and publisher software"
```
Comment (kadhonn) [inline boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/controller/StartPageController.kt:17]: hmm
do we want to focus on events or on our software here? i think the former, so i would focus on that

something mentioning more our loval events, or accessibility or something like that

Comment (mahdikhashan) [reply boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/controller/StartPageController.kt:17]: Boudicca.event - find accessible events in austria ?

Comment (kadhonn) [reply boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/controller/StartPageController.kt:17]: yeah something like that! :)

File: boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/controller/StartPageController.kt [hunk 2]
```diff
@@ -14,7 +14,7 @@ import org.springframework.web.servlet.ModelAndView
 @RequestMapping("/")
 class StartPageController @Autowired constructor(private val eventService: EventService) {
 
-    private val PAGE_TITLE = "Boudicca.Events - Open source event collector and publisher software"
+    private val PAGE_TITLE = "Boudicca.Event - find accessible events in Austria"
```
Comment (kadhonn) [inline boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/controller/StartPageController.kt:17]: the page is still called "Boudicca.Events", but the additional text looks fine for me

Comment (mahdikhashan) [reply boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/controller/StartPageController.kt]: done

**Review by kadhonn — APPROVED** [review]

**Pull request merged**